packages/react-liveness/src/components/FaceLivenessDetector/displayText.ts

```diff
@@ -5,7 +5,7 @@ export type HintDisplayText = {
   hintCanNotIdentifyText?: string;
   hintTooCloseText?: string;
   hintTooFarText?: string;
-  hintHoldFacePositionCountdownText?: string;
+  hintHoldFacePositionCountdownText?: string /** @deprecated String is no longer in use */;
```

Comment:
does this show in intellisense?

Reply:
Fixed this and now it shows correctly in intellisence